src/main/java/com/okta/developer/security/oauth2/ScimUserService.java

```diff
@@ -132,6 +133,7 @@ private ScimUser saveUserFromIdP(ScimUser scimUser) {
         User user = new User();
         user.setId(scimUser.getId());
         user.setEmail(scimUser.getPrimaryEmailAddress().get().getValue());
+        user.setLogin(user.getEmail().toLowerCase().toLowerCase(Locale.ROOT));
```

Comment:
There is an ugly line below that calls `getUserWithAuthoritiesByLogin` and uses this `login` value.  The OIDC user ID isn't available in the SCIM payload 😢